Initial setup

renangabriel27 opened the pull request.

dmarczal commented:
Itens para revisar
- Atualizar a versão do ruby para 2.5.1 
- Remover gems desnecessárias:
  - jbuilder
  - sqlite3
- Separar as gems adicionadas das gems padrões do rails pulando uma linha
- Atualizar o README com uma descrição simples do projeto.
- Sugiro apagar os comentários do spec/rails_helper.rb e do spec/spec_helper.rb

Comentários
  - Pensei em usar ECMAScript 6 e não usar coffescript, o que acha? Ver as vantagens.
  - Pensar em apresentar todo texto das páginas por meio do I18N.

renangabriel27 commented:
- Updated ruby version and Readme 
- Removed some gems and deleted comments of spec/

Sure, I'll use ECMAScript 6 in this project and the i18n gem for all texts to facilitate the internationalization.
